src/main/java/clara/benchmark/interop/AccumRepeatedlyNotSelected.java

```diff
@@ -0,0 +1,50 @@
+package clara.benchmark.interop;
+
+import clara.rules.RuleLoader;
+import clara.rules.WorkingMemory;
+import clara.rules.QueryResult;
+import clara.benchmark.beans.Customer;
+import clara.benchmark.IBenchmark;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.lang.Iterable;
+
+public class AccumRepeatedlyNotSelected implements IBenchmark<WorkingMemory> {
+
+    //@Override
+    public String getName() {
+        return "clara.accum-repeatedly-not-selected";
+    }
+
+    //@Override
+    public WorkingMemory getSession() throws Exception {
+        return RuleLoader.loadRules("clara.benchmark.accum-repeatedly-not-selected");
+    }
+
+    //@Override
+    public Iterable<?> getFacts() throws Exception {
+        List<Customer> customers = new ArrayList<Customer>();
+        for (int i = 1; i < 100; i++) {
+            customers.add(new Customer(i, "CUSTOMER_NAME", true));
+        }
+        return customers;
+    }
+
+    //@Override
+    public void run(WorkingMemory session, Iterable<Object> facts) throws Exception {
+        WorkingMemory lastSession = session;
+
+        for (Object fact : facts) {
+            List<Object> toInsert = new java.util.ArrayList<Object>();
```

Comment:
I suppose that is fine to test it the way you are in terms of insert fact + fire every iteration.

I don't think that is a regular case, but if you think it shows the behavior you are aiming for that's fine for now.